docs/reference/contributing/target/hw-accelerated-crypto.md

```diff
@@ -20,7 +20,7 @@ The Mbed TLS library was written in C and it has a small amount of hand-optimize
 
 ##### What parts can I accelerate?
 
-Mbed TLS has separate modules for the different cryptographic primitives. Hardware acceleration interface is available for the following modules and functions:
+Mbed TLS has separate modules for the different cryptographic primitives, each containing multiple function building blocks. Hardware acceleration interfaces are available both on the module and the function level. In the following list, every module listed can be replaced as a whole, and any function listed can alternatively be replaced on its own:
```

Comment:
It is already there, but maybe we could emphasise more that module and function level replacement are mutually exclusive.

Reply:
Query: Can we replace "every module listed can be replaced as a whole" with "every whole module can be replaced"? Also, who or what replaces modules and functions?

Reply:
I feel that the original sentence better conveys what we are trying here. What we want to emphasise that in this case they have to implement and plug in the whole functionality of the module and not just fragments of it.

The reader is likely to be an engineer who want to replace parts of TLS with their own implementation, for example because they have a hardware accelerator.